tests/features/security/xss_vulnerability.feature

```diff
@@ -32,3 +32,17 @@ Feature: Xss Vulnerability
     And I click "Article with xss Code in an image"
     And the response should contain "&lt;script&gt;alert('xss alert');&lt;/script&gt;"
     And the response should not contain "<script>alert('xss alert');</script>"
+
+   Scenario: Test the "Sanitize HTML" filter is never disabled if the "Display any HTML as plain text" one is in a "text format"
+     When I go to "/admin/config/content/formats/filtered_html"
+     And I uncheck the box "Sanitize HTML"
+     And I press "Save configuration"
+     Then I should see the success message "The text format Filtered HTML has been updated."
+     And the "Sanitize HTML" checkbox should be checked
+     When I go to "admin/config/content/formats/plain_text"
+     And the "Display any HTML as plain text" checkbox should be checked
```

Comment:
To better understand the syntax of the behat test, I think that:
If you want to use "the "Display any HTML as plain text" checkbox should be checked", you need to use the "Then" prefix.
But if you want to use the "And" prefix, it's better to use the sentence "the checkbox "(Display any HTML as plain text" is checked".

Reply:
Okido